src/components/LandDetailPage/LandDetailPage.tsx

```diff
@@ -146,7 +149,12 @@ export default class LandDetailPage extends React.PureComponent<Props, State> {
                             text={t('land_detail_page.set_operator')}
                             onClick={() => onNavigate(locations.landOperator(land.id))}
                           />
-                          <Dropdown.Item text={t('land_detail_page.set_link')} onClick={() => onNavigate(locations.landEns(land.id))} />
+                          {FF_ENS ? (
```

Comment:
Sorry I saw this one second, but could we change it to look the same as the one juanca did? here https://github.com/decentraland/builder/pull/1151/files#diff-e6f76a45490df7734399812bad8e19dda9c880883a47f3101bf25025124d65afR20

it's super minor, but for consistency sake